app/src/index.js

```diff
@@ -1,18 +1,33 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+import { Auth0Provider } from "@auth0/auth0-react";
+import { BrowserRouter as Router } from "react-router-dom";
+
 import "./index.css";
 import App from "./App";
-import reportWebVitals from "./reportWebVitals";
+//import reportWebVitals from "./reportWebVitals";
+
+// const domain = "dev-m1so.us.auth0.com";
+// const clientID = "BTb7r8IKJVu9BnKFll6N19bjBfa8heqw";
+// const auth0CallbackUrl = "http://localhost:3000/"; // window.location.origin
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Auth0Provider
+      domain="dev-m1so.us.auth0.com"
+      clientId="BTb7r8IKJVu9BnKFll6N19bjBfa8heqw"
+      redirectUri={window.location.origin}
+    >
```

Comment:
we probably will need to put those in .env file later, as suggested in the blog post 😬